examples/resources/adfs-connector.yaml

```diff
@@ -50,8 +50,8 @@ spec:
 
   attributes_to_roles:
     - name: "http://schemas.xmlsoap.org/claims/Group"
-      value: "teleadmins"
-      roles: ["admins"]
+      value: "admins"
```

Comment:
Might want to make this "Administrators" so it's more obvious that it's referring to a Windows group name

Reply:
ty. I didn't know that. I've updated it.